.github/actions/verify-submodule-pull-request/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Verify Submodule Pull Request
+description: >
+  Assuming that the current repository is a submodule in another repository, this action verifies that a pull request 
+  with a matching issue code has been opened there as well.
+
+inputs:
+  title:
+    required: true
+    description: The title of the current pull request.
+  repository:
+    required: false
+    default: Lombiq/Open-Source-Orchard-Core-Extensions
+    description: The parent repository's full name in "{owner}/{repository}" format.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup and Print PR Title
+      shell: pwsh
+      run: |
+        echo "Pull Request Title: ${{ inputs.title }}"
+
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+
+        New-Item -ItemType File -Force $Profile | Out-Null
+        Get-Content -Raw ${{ github.action_path }}/functions.ps1 >> $Profile
+
+    - name: Ensure Current PR Title Includes Issue Code
+      shell: pwsh
+      run: Check-Current '${{ inputs.title }}'
```

Comment:
How come this works without `${{ github.action_path }}`?

Reply:
Because I have added `${{ github.action_path }}` to the `$PATH` [here](https://github.com/Lombiq/GitHub-Actions/pull/19/files/360315f11903dd418d7dc0c0105f8d3bbc6faff6#diff-11fc0ebf975145d2e555831bbefb3f686c3bc46455c94e5fe1bb148d7f4785a2R23). 😏

Reply:
Ah, cool technique! Added to the wiki.